admin/admin-service/src/main/java/io/mosip/admin/config/Mapper.java

```diff
@@ -156,7 +156,7 @@ private static void loadEntity() {
 		entityMap.put("doc_category", DocumentCategory.class);
 		entityMap.put("doc_type", DocumentType.class);
 		entityMap.put("dynamic_field", DynamicField.class);
-		entityMap.put("reg_exceptional_holiday", ExceptionalHoliday.class);
+		//entityMap.put("reg_exceptional_holiday", ExceptionalHoliday.class);

```

Comment:
Why is this removed ?

Reply:
two entries with same key different values, internally first value will be replaced with reassigned value hence commented this.